transport/server.cc

```diff
@@ -668,7 +668,7 @@ future<> cql_server::connection::process_request() {
             ++_server._stats.requests_shed;
             return _read_buf.skip(f.length).then([this, stream = f.stream] {
                 const char* message = "request shed due to coordinator overload";
-                clogger.debug("{}: {}, stream {}", _client_state.get_remote_address(), message);
+                clogger.debug("{}: {}, stream {}", _client_state.get_remote_address(), message, stream);
```

Comment:
I thought compile-time format string checking is supposed to capture such things, does it?

Reply:
yes, i am still struggling with https://github.com/scylladb/seastar/pull/1196 and https://github.com/scylladb/seastar/pull/1727 . this is what i got from the former.